src/routes/release.js

```diff
@@ -197,15 +197,15 @@ router.get(
       return res.redirect('/');
     }
 
-    let releaseNotes = release.body;
-    const parsed = semver.parse(version);
-    if (parsed.prerelease.length) {
-      releaseNotes = releaseNotes.split(new RegExp(`@${escapeRegExp(version.substr(1))}\`?.`))[1];
-    }
-    releaseNotes =
-      '# Release Notes\n' +
-      releaseNotes.replace(/# Release Notes for [^\r\n]+(?:(?:\n)|(?:\r\n))/i, '');
+    let releaseNotes = release && release.body ? release.body : 'No release notes available.';
+
+const parsed = semver.parse(version);
+if (parsed.prerelease.length) {
+  releaseNotes = releaseNotes.split(new RegExp(`@${escapeRegExp(version.substr(1))}\`?.`))[1] || releaseNotes;
+}
 
+//  replace() now only runs if releaseNotes is valid
+releaseNotes = '# Release Notes\n' + (releaseNotes ? releaseNotes.replace(/# Release Notes for [^\r\n]+(?:(?:\n)|(?:\r\n))/i, '') : '');
```

Comment:
We know `releaseNotes` will be defined as of this point so we don't need this check here either.